tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
Any reason for sticking with this instead of the `git ls-files | grep '\.md$'` approach? I kinda like that this git method will automatically pick up any new folders that we may add in the future but don't feel strongly